specify/specify7
# Add a way to explicity choose aggregators in QB

Fixes #4620

### Checklist

      and self-explanatory (or properly documented)

### Testing instructions

- Navigate to the Query Builder (QB).
- Select a table that you know has one or several aggregators (To check, go to app resources, navigate to Record Formatter, choose the table, select aggregator tab, and verify the list of aggregators names available).
- Choose "aggregated."

- [ ] Verify that in the query line, after selecting the "aggregated" field box, a drop-down appears with a list of aggregators choices (corresponding to the list in app resources).

 Verify that changing the aggregator changes the way the field is displayed in the results table.
